### PR TITLE
docs: dibbler is live update 70 

### PR DESCRIPTION
**Pull request by delta1**
dibbler release post

### CODE REVIEW

File: _updates/2022-01-28-update-70.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+thumbnail: update-background.jpg
+title: Dibbler testnet is live
+subtitle: The latest Tari testnet has landed!
+class: subpage
+permalink: /dibbler-testnet-is-live/
+---
+
+This week the Tari development community rolled out the newest iteration of the Tari testnet. It is lovingly named [Dibbler], and brings a host of upgrades.
+
+There is also incredible and exclusive [new drip] in the TTL store. Check out the new [mining tutorials] to earn yours!
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md]:
```suggestion
There is also incredible and exclusive [new swag] in the TTL store. Check out the new [mining tutorials] to earn yours!
```

File: _updates/2022-01-28-update-70.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Another new application is Tari Collectibles. This is a cross-platform desktop application that will allow Asset Issuers to register their new assets and tokens, and mint NFTs on their asset sidechains.
+
+### Aurora Yat Integration
+
+Last but certainly not least, Tari [Aurora] has been polished up and now supports native [Yat] integration! This makes it even easier to send and receive Tari when you link up your Yat to your Aurora wallet. Check out the other [updates] in `v0.10` now!
+
+[dibbler]: https://discworld.fandom.com/wiki/Cut-Me-Own-Throat_Dibbler
+[tari codebase]: https://github.com/tari-project/tari
+[mining tutorials]: {% link _updates/2022-01-24-update-69.md %}
+[new drip]: https://store.tarilabs.com
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md]:
```suggestion
[new swag]: https://store.tarilabs.com
```

File: _updates/2022-01-28-update-70.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+This week the Tari development community rolled out the newest iteration of the Tari testnet. It is lovingly named [Dibbler], and brings a host of upgrades.
+
+There is also incredible and exclusive [new drip] in the TTL store. Check out the new [mining tutorials] to earn yours!
+
+In the 6 months since Weatherwax launched in July 2021, the [Tari codebase] has seen a tremendous amount of work: 14 Contributors submitted 649 PRs consisting of new features, new applications, bug fixes, tests and other changes.
+
+Let's take a look at some of the most important enhancements.
+
+<iframe src="https://giphy.com/embed/a8GOUGGMCOSIM" width="480" height="320" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md]: We don't need an iFrame for a giphy, surely!

File: _updates/2022-01-28-update-70.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+### Asset Registration
+
+The structure of transaction outputs has been upgraded to enable non-fungible tokens. These outputs will register asset and token classes on the base layer, defining their metadata and validator committees.
+
+These changes also enable the asset checkpoint functionality that will be used to record the periodic state of assets in all layer 2 sidechains.
+
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md:28]:
```suggestion

The features are still an early proof-of-concept and will be improved and streamlined over the next several releases.
```

File: _updates/2022-01-28-update-70.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+### Covenants
+
+Covenants are a critical part of the digital asset picture. They allow the creator of an output to specify rules that will be kept and applied to outputs created from spending it. This enables functionality like ongoing royalties on asset sales and transfers.
+
+### New Emission Curve
+
+Dibbler also comes complete with a new emission curve! This means the size of coinbase payouts are about 3x as much as they were in the previous testnet. With a target time of 2 minute blocks, the new emission curve aims to supply [21 billion Tari], with the first half of that being supplied in the first 3 years. The tail emission has been set to 800 Tari, which works out to roughly 1% inflation after the initial supply has been mined.
+
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md:37]: This is not necessarily what the mainnet emission schedule will look like, but the developer community is leaning towards proposing something akin to this.

File: _updates/2022-01-28-update-70.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+These changes also enable the asset checkpoint functionality that will be used to record the periodic state of assets in all layer 2 sidechains.
+
+### Covenants
+
+Covenants are a critical part of the digital asset picture. They allow the creator of an output to specify rules that will be kept and applied to outputs created from spending it. This enables functionality like ongoing royalties on asset sales and transfers.
+
+### New Emission Curve
+
+Dibbler also comes complete with a new emission curve! This means the size of coinbase payouts are about 3x as much as they were in the previous testnet. With a target time of 2 minute blocks, the new emission curve aims to supply [21 billion Tari], with the first half of that being supplied in the first 3 years. The tail emission has been set to 800 Tari, which works out to roughly 1% inflation after the initial supply has been mined.
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md]:
```suggestion
Dibbler also comes complete with a new emission curve! This means the size of coinbase payouts are about 3x as much as they were in the previous testnet with a Monero-like schedule and tail emission. The base emission is [21 billion testnet Tari]. The tail emission has been set to 800 Tari, which works out to roughly 1% inflation after the initial supply has been mined.
```

File: _updates/2022-01-28-update-70.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+### Validator Node
+
+One of the new applications released in Dibbler, the committees of validator nodes will take on consensus duties in the various asset sidechains. They enable assets to be sold and transferred on the second layer, while committing their state periodically back to the Tari blockchain. Validator node is still a work in progress, with a lot more layer 2 goodies coming soon.
```
Comment (CjS77) [inline _updates/2022-01-28-update-70.md]:
```suggestion
One of the new applications released in Dibbler, the committees of validator nodes will take on consensus duties in the various asset side-chains. They enable assets to be sold and transferred on the second layer, while committing their state periodically back to the Tari blockchain. Validator node is still a work in progress, with a lot more layer 2 goodies coming soon.
```

**Review by CjS77 — COMMENTED** [review]
Thanks! A few editorial comments.

File: _updates/2022-01-28-update-70.md (hunk 3 above)
Comment (delta1) [reply _updates/2022-01-28-update-70.md]: lol I just copied their embed 🤷‍♂️

File: _updates/2022-01-28-update-70.md (hunk 1 above)
Comment (delta1) [reply _updates/2022-01-28-update-70.md]: Drip is the new swag!

**Review by CjS77 — APPROVED** [review]
Cool

**Pull request merged**